docs/specification.md

```diff
@@ -208,17 +211,21 @@ Retrieves the current state (including status, artifacts, and optionally history
 **Inputs:**
 
 - `taskId`: Unique identifier of the task to retrieve
-- `historyLength` (optional): Number of recent messages to include in the task's history (see [History Length Semantics](#332-history-length-semantics) for details)
+- `historyLength` (optional): Number of recent messages to include in the task's history (see [History Length Semantics](#323-history-length-semantics) for details)
 
 **Outputs:**
 
 - [`Task`](#411-task): Current state and artifacts of the requested task
 
+**Errors:**
+
+None specific to this operation beyond standard protocol errors.
```

Comment:
Maybe we should call out things like input validation errors? i.e. invalid historyLength (negative, or not an integer)

Reply:
I updated section 3.3.2 to try and be clearer about how to handle standard error conditions like invalid parameters. I'd rather not try and enumerate every possible error on every operation.  The history length semantics section clarifies the valid values of the history length parameter.